polkadot/node/subsystem-bench/src/lib/approval/mod.rs

```diff
@@ -841,15 +835,22 @@ fn build_overseer(
 pub fn prepare_test(
 	config: TestConfiguration,
 	options: ApprovalsOptions,
+	with_prometheus_endpoint: bool,
```

Comment:
I don't like it and I think I should start the endpoint outside the test, like we do with pyroscope. It can be done in next PRs.